packages/fx-core/src/plugins/resource/frontend/dotnet/plugin.ts

```diff
@@ -77,6 +90,56 @@ export class DotnetPluginImpl {
     throw new FetchConfigError(key);
   }
 
+  public async scaffold(ctx: PluginContext): Promise<TeamsFxResult> {
+    return ok(undefined);
+  }
+
+  public async generateArmTemplates(ctx: PluginContext): Promise<TeamsFxResult> {
+    const bicepTemplateDirectory = PathInfo.bicepTemplateFolder(getTemplatesFolder());
+
+    const provisionTemplateFilePath = path.join(bicepTemplateDirectory, Bicep.ProvisionFileName);
+    const provisionWebappTemplateFilePath = path.join(
+      bicepTemplateDirectory,
+      WebappBicepFile.provisionTemplateFileName
+    );
+
+    const configTemplateFilePath = path.join(bicepTemplateDirectory, Bicep.ConfigFileName);
+    const configWebappTemplateFilePath = path.join(
+      bicepTemplateDirectory,
+      WebappBicepFile.configurationTemplateFileName
+    );
+
+    const solutionSettings = ctx.projectSettings!.solutionSettings as AzureSolutionSettings;
+    const plugins = getActivatedV2ResourcePlugins(solutionSettings).map(
+      (p) => new NamedArmResourcePluginAdaptor(p)
+    );
+    const pluginCtx = { plugins: plugins.map((obj) => obj.name) };
+
+    const provisionOrchestration = await generateBicepFromFile(
+      provisionTemplateFilePath,
+      pluginCtx
+    );
+    const provisionModule = await generateBicepFromFile(provisionWebappTemplateFilePath, pluginCtx);
+    const configOrchestration = await generateBicepFromFile(configTemplateFilePath, pluginCtx);
+    const configModule = await generateBicepFromFile(configWebappTemplateFilePath, pluginCtx);
+    const result: ArmTemplateResult = {
+      Provision: {
+        Orchestration: provisionOrchestration,
+        Modules: { webapp: provisionModule },
+      },
+      Configuration: {
+        Orchestration: configOrchestration,
+        Modules: { webapp: configModule },
+      },
+      Reference: {
+        webappResourceId: WebappBicep.webappResourceId,
+        endpoint: WebappBicep.webappEndpoint,
+        domain: WebappBicep.webappDomain,
+      },
+    };
+    return ok(result);
+  }
+
   public async postProvision(ctx: PluginContext): Promise<TeamsFxResult> {
     return ok(undefined);
```

Comment:
Do we have any post provision step for dotnet plugin?

Reply:
No. Just to align with FrontendPluginImpl.